gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.1-bin.zip
```

Comment:
Updating the Gradle wrapper did not change the JAR?

Reply:
./gradlew wrapper --gradle-version 7.4.1 command was executed and Gradle wrapper JAR was updated